build/cmake/functions.cmake

```diff
@@ -324,6 +324,11 @@ function(wx_set_target_properties target_name)
             )
     endif()
 
+    if(WIN32)
+        target_compile_definitions(${target_name} PUBLIC UNICODE)
+    endif()
+    target_compile_definitions(${target_name} PUBLIC _UNICODE)
```

Comment:
Do we really need to predefine this under non-Win32 systems? I don't think it's used there, am I missing something?

Reply:
It will be used on any platform. When you link with another target in CMake, your target will inherit the public headers, libraries, and defines.

Reply:
Yes, sorry, I understand this, what I don't understand is why should the projects using wx get this define under non-Win32 platforms when it's only used by (MSVC) CRT under Win32 AFAIK.

Reply:
I'm still not sure about this but if you're confident this is correct, we could merge this, of course. Please let me know if we should?

Reply:
No, you are right. I confused the msvc c- runtime with the c++ library. I'll make it win32 only.